### DISCUSSION TITLE
Running a container with podman-py API with --dns=none and --no-hosts

### DISCUSSION THREAD

**Discussion by th-hummel** [containers/podman#23178] [Q&A]
Hello,

Is it possible using `podman-py` API to run an image with the same semantics as if we ran it with `podman`  `--dns=none` and `--no-hosts` cmdline options ?

Passing `dns=[]` still got me `resolv.conf` bind mounted (hence managed by podman)
and I did not see any option to tell not to manage `/etc/hosts`

Thanks for your help

-- 
Thomas HUMMEL

**Comment by rhatdan** [comment]
@umohnani8 @jwhonce PTAL
